### PR TITLE
Look for Colab badges and redirect them to master

### LINKED ISSUE #81: Colab links are broken / don't point to master
When you pull a colab from somewhere else, the link that colab auto-adds at the top of the ipython notebook links to the specific branch that the colab was saved to, not to master. Example:

https://github.com/NeuromatchAcademy/course-content/pull/80/files

One points to the day06 branch, the other to Madineh's GH.

### PR DESCRIPTION
**Pull request by mwaskom**
Fixes #81

### CODE REVIEW

**Comment by mwaskom** [comment]
Fingers crossed that we're not in "now we have two problems" territory here.

File: ci/process_notebooks.py [hunk 1]
```diff
@@ -157,6 +163,18 @@ def has_solution(cell):
...
 
+def has_colab_badge(cell):
+    """Return True if cell has a Colab badge as an HTML element."""
+    return "colab_badge.svg" in cell["source"]
+
+
+def redirect_colab_badge(cell):
+    """Modify the Colab badge to point at the master branch on Github."""
+    cell_text = cell["source"]
+    p = re.compile(r"^(.+/NeuromatchAcademy/course-content/blob/)\w+(/.+$)")
```
Comment (patrickmineault) [inline ci/process_notebooks.py:174]: I think this wouldn't work if someone pulls in from a fork.

Comment (mwaskom) [reply ci/process_notebooks.py:174]: That's true. But is that the workflow we're going to use? I'm not sure this bit of processing can solve all problems — we'd also have to replace the repository name in the badge then too.

Canonical Colab links should probably be hand-curated in the README / on the website.

**Comment by mwaskom** [comment]
Let's get this merged for now since it works for the workflow we currently prescribe. We can keep an open issue about broken badges for colabs that come in from a feature on a fork.

**Pull request merged**